src/components/page-layout/Layout.js

```diff
@@ -12,34 +12,37 @@ import { useStaticQuery, graphql } from 'gatsby';
 // Components
 import Header from './Header';
 import NavigationBar from './NavigationBar';
+import GrocerySection from '../grocery-view/grocery-section/GrocerySection';
 
 // Styles
 import styles from './styles/layout.module.css';
 
 const Layout = ({ children }) => {
   const data = useStaticQuery(graphql`
-        query SiteTitleQuery {
-            site {
-                siteMetadata {
-                    title
-                }
-            }
+    query SiteTitleQuery {
+      site {
+        siteMetadata {
+          title
         }
-    `);
+      }
+    }
+  `);
 
   return (
-        <>
-            <Header siteTitle={data.site.siteMetadata.title} />
-            <NavigationBar />
-            <div className={styles.body}>
-                <main>{children}</main>
-                <footer>
-                    © {new Date().getFullYear()}, Built with
-                    {' '}
-                    <a href="https://www.gatsbyjs.org">Gatsby</a>
-                </footer>
-            </div>
-        </>
+    <>
+      <Header siteTitle={data.site.siteMetadata.title} />
+      <NavigationBar />
+      <div className={styles.body}>
+        <main>
+          {children}
+          <GrocerySection />
+        </main>
+        <footer>
+          © {new Date().getFullYear()}, Built with{' '}
+          <a href="https://www.gatsbyjs.org">Gatsby</a>
+        </footer>
```

Comment:
Do we need this footer for any reason?

Or maybe we just want to replace it with

```
<footer>
©2020, A <a href="https://www.viget.com/">Viget</a> Project
Liam Becker, Jennifer Montoya, William Dinneen, Jackson Doyle, Mika Byar
</footer>
```

Or something like that